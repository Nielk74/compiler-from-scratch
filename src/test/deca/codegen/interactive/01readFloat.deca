// Description:
// Test du readFloat dans l'initialisation et l'assignation.
// On déclare deux variables, x sans initialisation,
// et y avec une initialisation avec un readFloat.
// On assigne ensuite à x une valeur avec un readFloat.
//
//
// Historique:
//    crée le 10/06/2022
{
    float x;
    float y = readFloat();
    x = readFloat();
    println(x);
    println(y);
}
