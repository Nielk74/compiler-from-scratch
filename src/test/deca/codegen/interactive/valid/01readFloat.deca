// Description:
// Test du readFloat dans l'initialisation et l'assignation.
// On déclare deux variables, x sans initialisation,
// et y avec une initialisation avec un readFloat.
// On assigne ensuite à x une valeur avec un readFloat.
//
// Resultats:
//  6.78900e+01
//  5.60000e+00
//
// Historique:
//    crée le 10/06/2022
{
    float x;
    float y = readFloat();
    x = readFloat();
    println(x);
    println(y);
}
