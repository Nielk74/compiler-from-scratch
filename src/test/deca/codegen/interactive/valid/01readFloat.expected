6.78900e+01
5.60000e+00
